extensions/standard-processors/tests/unit/ProcessGroupTestUtils.h

```diff
@@ -0,0 +1,288 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "TestBase.h"
+#include "YamlConfiguration.h"
+#include "Utils.h"
+
+struct Lines {
+  std::vector<std::string> lines;
+
+  std::string join(const std::string& delim) const {
```

Comment:
this seems to be the same as `StringUtils::join(const std::basic_string<TChar>& separator, const U& container)`

Reply:
done